reactioncommerce/reaction-next-starterkit
# feat: Add structured data markup to PDP

Resolves #149 
Impact: **minor**
Type: **feature**

## Issue
For SEO purposes, we should add structured data markup to the PDP page

## Solution
Use [`jsonld`](https://json-ld.org/) style to add structured data to the PDP. I think this is a good jumping off point, and we can always add more data in the future as it becomes available to us (reviews, etc)

## Breaking changes
None

## Testing
- Add a tool to test structured data on localhost pages (I installed this: https://chrome.google.com/webstore/detail/structured-data-testing-t/kfdjeigpgagildmolfanniafmplnplpl/related?hl=en)
- See that PDP has data provided to the markup tool
